components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -3,19 +3,34 @@ import { render, screen } from '@testing-library/react';
 import { WeeklyPickButton } from './WeeklyPickButton';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
 
+const weeklyPickData = {
+  team: 'Ravens',
+  src: '/path/to/ravens.svg',
+  height: '48',
+  width: '48',
+  alt: 'Ravens',
+  className:
+    'text-base font-normal leading-none text-zinc-50 cursor-pointer flex gap-2 items-center rounded-xl border-2 border-zinc-800 peer-aria-checked:border-orange-600 py-4 px-3 w-full peer-hover:bg-zinc-800 transition',
+};
+
 describe('WeeklyPickButton', () => {
   it('renders correctly', () => {
     render(
       <RadioGroup>
-        <WeeklyPickButton team="Test Team" src="/path/to/image.svg"/>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
       </RadioGroup>,
     );
+    const image = screen.getByRole('img', { name: /Ravens/i });
 
-    const image = screen.getByRole('img', { name: /Test Team/i });
     expect(image).toBeInTheDocument();
-    expect(image).toHaveAttribute('src', '/path/to/image.svg');
+    expect(image).toHaveAttribute('src', weeklyPickData.src);
 
-    const teamName = screen.getByText(/Test Team/i);
+    const teamName = screen.getByText(/Ravens/i);
     expect(teamName).toBeInTheDocument();
+    expect(teamName).toHaveClass(weeklyPickData.className);
```

Comment:
m: Never find an element by these long tailwind classes. This is very easy to break. You setup a data-testid that can be used already.